sycl/examples/simple-dpcpp-app.cpp

```diff
@@ -0,0 +1,54 @@
+#include <CL/sycl.hpp>
+
+using namespace cl::sycl;
+
+int main() {
+  // Create a buffer of 4 ints to be used inside the kernel code.
+  buffer<cl_int, 1> Buffer(4);
```

Comment:
```suggestion
  buffer<int, 1> Buffer(4);
```
Although the comment above isn't precise, I think it means C++ types, not OpenCL types.